### PR TITLE
Extracts and displays uniform title if available

### PR DESCRIPTION
**Pull request by JPrevost**
What:

* Extracts and displays uniform title if available for book and
  article results

Why:

* The uniform title often has important information to convey

https://mitlibraries.atlassian.net/browse/DI-113

### CODE REVIEW

**Comment by frrrances** [comment]
I'm seeing some funky results: 

screen shot 2017-05-01 at 11 54 54 am

screen shot 2017-05-01 at 11 53 33 am

Anything we can do about those cases?

File: app/assets/stylesheets/_results.scss [hunk 1]
```diff
@@ -47,6 +47,12 @@
     }
   }
 
+  .result-uniform_title {
```
Comment (frrrances) [inline app/assets/stylesheets/_results.scss]: looks nice but no underscores:
```
.result-title-uniform
```

File: app/views/search/_result.html.erb [hunk 2]
```diff
@@ -14,6 +14,11 @@
   <h3 class="result-title">
     <span class="sr">Title: </span><%= link_to(result.title, result.url, class: 'bento-link', data: {type: "Title"} ) %>
   </h3>
+  <div class="result-uniform_title">
```
Comment (frrrances) [inline app/views/search/_result.html.erb]: no underscores: ``.result-title-uniform``

File: app/assets/stylesheets/_results.scss (hunk 1 above)
Comment (JPrevost) [inline app/assets/stylesheets/_results.scss]: One day we will discuss this at length. I will lose, but I want to say my words about it :)

**Comment by JPrevost** [comment]
I'll take a look at those oddly encoded displays. I think it's either a quick fix or "oh god what did they do".

File: app/assets/stylesheets/_results.scss (hunk 1 above)
Comment (frrrances) [inline app/assets/stylesheets/_results.scss]: deal. we will fight. 🔪

**Comment by JPrevost** [comment]
It's definitely "oh god what did they do". I'll try to come up with a reasonably sane solution to what is definitely some awful data practices.

**Comment by frrrances** [comment]
😭

**Pull request merged**